packages/automerge-repo/src/synchronizer/DocSynchronizer.ts

```diff
@@ -158,4 +173,12 @@ export class DocSynchronizer extends Synchronizer {
       return newDoc
```

Comment:
Unrelated, but why doesn't the preceding call to `this.#sendSyncMessage` take `newDoc` instead of `doc`? Don't we want to sync the latest changes?

Reply:
Interesting. Do you mean the other way around? Looks like it should take `newDoc` to me

Reply:
I can see why this would still work, as `newDoc` only differs from `doc` with the changes from the peer, which of course the peer already has. Still doesn't look quite right to me though.